File: Baekjoon/Problem_25000~25999/Problem_25716/Problem_25716.cpp
#include <cstdio>

using namespace std;

int factorial(const int n) {
    int _prod = 1;
    for (int i = 1; i <= n; i++) _prod *= i;
    return _prod;
}

int main() {
    int N; scanf("%d", &N);
    printf("%d\n", factorial(N));
    return 0;
}

File: Baekjoon/Problem_25000~25999/Problem_25716/Problem_25716.py
import sys

input = lambda: sys.stdin.readline().rstrip()

def factorial(n: int) -> int:
    _prod = 1
    for i in range(1, n+1):
        _prod *= i
    return _prod

if __name__ == "__main__":
    N = int(input())
    
    print(factorial(N))